src/locales/ca/en/index.json

```diff
@@ -1,6 +1,8 @@
 {
-  "alreadyHaveAnAccount": "Already have an account? <a href='/'>Sign in here!</a>",
-  "dontHaveAnAccount": "Don’t have an account? <a href='/register'>Sign up here!</a>",
+  "alreadyHaveAnAccount": "Already have an account? <a href='?journey'>Sign in here!</a>",
+  "backToDefault": "Back to Sign In",
+  "backToLogin": "Back to Sign In",
+  "dontHaveAnAccount": "Don’t have an account? <a href='?journey=Registration'>Sign up here!</a>",
```

Comment:
My guess would be that `onMount()` in the test app isn't being called if the URL is just changed here instead of a proper page redirect, so the journey store isn't being initialized with the tree from the `journeyParam` - just my initial thought but maybe there's something I'm doing wrong? Although looking at the `match` array for the Registration journey, it now seems like we're expecting to capture that link navigation and turn it into an init of the journey store with that journey instead of following the link, but that isn't working.

Reply:
There was some logic in the linkWrapper that broke. It should now be fixed.